src/a01/models/run.py

```diff
@@ -91,7 +95,7 @@ def get_table_view(self) -> Generator[List, None, None]:
         for run in self.runs:
             time = (run.creation - datetime.timedelta(hours=8)).strftime('%Y-%m-%d %H:%M PST')
             remark = run.details.get('remark', None) or run.settings.get('a01.reserved.remark', '')
-            owner = run.details.get('creator', None) or run.details.get('a01.reserved.creator', '')
+            owner = run.owner or run.details.get('creator', None) or run.details.get('a01.reserved.creator', '')
 
```

Comment:
Is there a place where we could document changes in the schema?

Reply:
We probably can put it at the services' repo.